autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/selecting/AutofillSelectCredentialsGrouper.kt

```diff
@@ -33,6 +33,7 @@ interface AutofillSelectCredentialsGrouper {
     data class Groups(
         val perfectMatches: List<LoginCredentials>,
         val partialMatches: Map<String, List<LoginCredentials>>,
+        val otherGroups: Map<String, List<LoginCredentials>>,
```

Comment:
does it make sense to call this `sameCompanyMatches`? 

Edit: it seems you refer to them in other parts of the code as `shareableCrendentials`, so maybe something to use here.